guides/source/engines.md

```diff
@@ -1165,7 +1165,7 @@ module Blorgh::Concerns::Models::Article
 
   # 'included do' causes the included code to be evaluated in the
   # context where it is included (article.rb), rather than being
```

Comment:
What do you think about also changing this line to mention `Blorgh/app/models/blorgh/article.rb`?  That way we differentiate between the two `article.rb` files.

Alternatively, we could change both file paths to their respective Ruby modules (i.e. `Blorgh::Article` and `Blorgh::Concerns::Models::Article`).

Reply:
Sounds good to me. Fixed